frontend/src/static/js/main.js

```diff
@@ -133,6 +151,22 @@ window.app = new window.Vue({
 
     errorMessages: {},
   },
+  computed: {
+    zoomTabInfo() {
+      return this.$store.state.zoomTabInfo;
+    },
+    authorshipInfo() {
+      return this.$store.state.authorshipInfo;
+    },
+  },
+  watch: {
+    zoomTabInfo() {
```

Comment:
Can the component watch the state directly? In my memory, it can.  Meanwhile, if we are already watching Info, it may be better to put code of that the method inside the watch code body directly for simplicity.

Reply:
Do you mean put the updateTabZoom inside the watch method? The zoom one can but the authorship one is also used somewhere else so cannot shift.